pkg/registry/scheduling/priorityclass/storage/storage_test.go

```diff
@@ -40,7 +40,8 @@ func newStorage(t *testing.T) (*REST, *etcd3testing.EtcdTestServer) {
 		DeleteCollectionWorkers: 1,
 		ResourcePrefix:          "priorityclasses",
 	}
-	return NewREST(restOptions), server
+	rest, _ := NewREST(restOptions)
```

Comment:
doesn't make sense. the pull propagates the errors but just omits them in a different way.